### PR TITLE
Cleanup seek reception

### PR DESCRIPTION
**Pull request by Alberth289346**
Cleanup patches from #2281

### CODE REVIEW

**Review by lewri — APPROVED** [review]

File: CorsixTH/Lua/humanoid_actions/seek_reception.lua [hunk 1]
```diff
@@ -23,20 +23,44 @@ class "SeekReceptionAction" (HumanoidAction)
...
+  local pathindex = #path_x
+  for i = pathindex - 1, 2, -1 do
+    if can_join_queue_at(humanoid, path_x[i], path_y[i], pathindex - i) then
+      path_x[i + 1] = nil
+      path_y[i + 1] = nil
+    else
+      break
+    end
+  end
+  return
```
Comment (lewri) [inline CorsixTH/Lua/humanoid_actions/seek_reception.lua:60]: Is the return necessary here?

**Review by lewri — APPROVED** [review]
Long as queueing has been tested with this on an existing save this all seems fine.

File: CorsixTH/Lua/humanoid_actions/seek_reception.lua (hunk 1 above)
Comment (Alberth289346) [reply CorsixTH/Lua/humanoid_actions/seek_reception.lua:60]: Doesn't look like it

Comment (TheCycoONE) [reply CorsixTH/Lua/humanoid_actions/seek_reception.lua:60]: Plan to change it? Will merge as is otherwise.

**Comment by Alberth289346** [comment]
Please merge

**Pull request merged**